js/dtos/MeteorDto.js

```diff
@@ -0,0 +1,27 @@
+class MeteorDto {
+  constructor(data) {
+    this.startDate = specifyStartDate(data)
+    this.endDate = specifyEndDate(data)
+    this.count = data.count
+    this.wereDangerous = specifyWereDangerous(data)
+  }
+}
+
+const specifyStartDate = (data) => {
+  const startDate = data.startDate
+  const endDate = data.endDate
+  return startDate === undefined && endDate !== undefined ? endDate : startDate
```

Comment:
Not obvious way, to be honest. In case startDate is not presented, it would be better to set it as null, and then proceed it in service (use some default value):
```
const startDate = requestedStartDate ? new Date(requestedStartDate) : new Date();
```